crates/walrus-service/src/node.rs

```diff
@@ -976,11 +1007,11 @@ impl StorageNode {
             Some(factory) => Some(factory.create().await?),
             None => None,
         };
-        let (event_stream, next_event_index) = self
+        let (event_stream, event_cursor) = self
```

Comment:
Nit: since we are returning the full event_cursor now, maybe add a log entry to print it out. Maybe helpful for future debugging.